From 123ee865e05903fac15553d7c094c5aa52b0cda7 Mon Sep 17 00:00:00 2001
From: Rajat Srivastava <rajat.srivastava@nxp.com>
Date: Thu, 1 Feb 2018 18:08:37 +0530
Subject: [PATCH 124/129] ls1088a: qspi: Enable XIP mode above 16 MB addresses

Currently in LS1088A, XIP mode in QSPI works up to 16 MB
addresses. This patch enables QSPI support in XIP mode for
addresses above 16 MB as well.

Signed-off-by: Rajat Srivastava <rajat.srivastava@nxp.com>
---
 configs/ls1088aqds_qspi_SECURE_BOOT_defconfig |    1 +
 configs/ls1088aqds_qspi_defconfig             |    1 +
 configs/ls1088ardb_qspi_SECURE_BOOT_defconfig |    1 +
 configs/ls1088ardb_qspi_defconfig             |    1 +
 4 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/configs/ls1088aqds_qspi_SECURE_BOOT_defconfig b/configs/ls1088aqds_qspi_SECURE_BOOT_defconfig
index a36badf..88117de 100644
--- a/configs/ls1088aqds_qspi_SECURE_BOOT_defconfig
+++ b/configs/ls1088aqds_qspi_SECURE_BOOT_defconfig
@@ -24,6 +24,7 @@ CONFIG_E1000=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_FSL_CAAM=y
 CONFIG_FSL_DSPI=y
+CONFIG_QSPI_AHB_INIT=y
 CONFIG_FSL_LS_PPA=y
 CONFIG_HUSH_PARSER=y
 CONFIG_NETDEVICES=y
diff --git a/configs/ls1088aqds_qspi_defconfig b/configs/ls1088aqds_qspi_defconfig
index 96b3de7..d89f9f7 100644
--- a/configs/ls1088aqds_qspi_defconfig
+++ b/configs/ls1088aqds_qspi_defconfig
@@ -38,6 +38,7 @@ CONFIG_PCIE_LAYERSCAPE=y
 CONFIG_SYS_NS16550=y
 CONFIG_DM_SPI=y
 CONFIG_FSL_DSPI=y
+CONFIG_QSPI_AHB_INIT=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_XHCI_HCD=y
diff --git a/configs/ls1088ardb_qspi_SECURE_BOOT_defconfig b/configs/ls1088ardb_qspi_SECURE_BOOT_defconfig
index 4c18d94..75e1c1a 100644
--- a/configs/ls1088ardb_qspi_SECURE_BOOT_defconfig
+++ b/configs/ls1088ardb_qspi_SECURE_BOOT_defconfig
@@ -25,6 +25,7 @@ CONFIG_E1000=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_FSL_CAAM=y
 CONFIG_FSL_DSPI=y
+CONFIG_QSPI_AHB_INIT=y
 CONFIG_FSL_LS_PPA=y
 CONFIG_HUSH_PARSER=y
 CONFIG_NETDEVICES=y
diff --git a/configs/ls1088ardb_qspi_defconfig b/configs/ls1088ardb_qspi_defconfig
index 0f721ea..1f9451d 100644
--- a/configs/ls1088ardb_qspi_defconfig
+++ b/configs/ls1088ardb_qspi_defconfig
@@ -39,6 +39,7 @@ CONFIG_PCIE_LAYERSCAPE=y
 CONFIG_SYS_NS16550=y
 CONFIG_DM_SPI=y
 CONFIG_FSL_DSPI=y
+CONFIG_QSPI_AHB_INIT=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_XHCI_HCD=y
-- 
1.7.1

